.github/workflows/backwards-compatibility-check.yaml

```diff
@@ -17,5 +17,18 @@ jobs:
       - name: "Install dependencies"
         run: composer global require "roave/backward-compatibility-check:^8.2"
       - name: "Check for BC breaks"
+        if: github.actor != 'app/release-please'
         run: |
           ~/.composer/vendor/bin/roave-backward-compatibility-check --from=origin/main --format=github-actions
+      - name: Get Latest Release
+        if: github.actor == 'app/release-please'
+        id: latest-release
+        uses: pozetroninc/github-action-get-latest-release@master
+        with:
+          repository: ${{ github.repository }}
+      - name: "Check for BC breaks (Next Release)"
+        if: github.actor == 'app/release-please'
+        run: |
+          ~/.composer/vendor/bin/roave-backward-compatibility-check \
+              --from=${{ steps.latest-release.outputs.release }} \
+              --to=main --format=github-actions
```

Comment:
Note: we could rework this to be something like this instead:

```bash
    FROM_REF="--from=origin/main"
    TO_REF=""
    if [ '${{github.actor}}' -eq 'app/release-please' ]; then
            FROM_REF="--from=${{ steps.latest-release.outputs.release }}"
            TO_REF="--to=origin/main"
    fi
    ~/.composer/vendor/bin/roave-backward-compatibility-check $FROM_REF $TO_REF
```

I am not sure if this is simpler or more complicated though.